torch/_numpy/_funcs_impl.py

```diff
@@ -1387,13 +1387,17 @@ def _sort_helper(tensor, axis, kind, order):
 
 
 def sort(a: ArrayLike, axis=-1, kind=None, order: NotImplementedType = None):
+    if a.dtype.is_complex:
+        return NotImplemented
     # `order` keyword arg is only relevant for structured dtypes; so not supported here.
     a, axis, stable = _sort_helper(a, axis, kind, order)
     result = torch.sort(a, dim=axis, stable=stable)
     return result.values
 
 
 def argsort(a: ArrayLike, axis=-1, kind=None, order: NotImplementedType = None):
+    if a.dtype.is_complex:
+        return NotImplemented
```

Comment:
Not: Can move to _sort_helper to remove duplication.

Reply:
but would need to return somehow from the main function, so I didn't really know how to do it uniformly. I figured we could always just raise a `NotImplementedError` and then catch it in the normaliser and then return...

Reply:
Given that, a the moment, we always raise, we should probably go with the normaliser approach.

The point as to why doing the return rather than the exception is that dynamo already knows how to treat a function that returns `NotImplemented`, falling back gracefully to eager.